packager/app/test/packager_test.py

```diff
@@ -781,6 +786,14 @@ def testDashOnlyAndHlsOnly(self):
         self._GetFlags(output_dash=True, output_hls=True))
     self._CheckTestResults('hls-only-dash-only')
 
+  def testDashLabel(self):
+    streams = [
+        self._GetStream('video', dash_label='English'),
```

Comment:
Can you please change this to something more realistic?  We wouldn't have a language label on a video stream.  It would also be better for the test to show that two separate labels are applied in the correct place, or that a label is only set if provided.  Otherwise, there could be a bug that the last label is applied to all streams.

Reply:
done